docs/slim/prune/prune.md

```diff
@@ -64,26 +66,26 @@ python slim/prune/prune.py \
        --save_dir prune_model
 ```
 
-## 部署
+## Deployment
 
-通过`剪枝`得到的模型，我们可以直接进行部署应用，相关教程请参考[模型部署](../../model_export.md)。
+The pruned model can be deployed directly. Please refer to the [tutorial](../../model_export.md) for model deployment.
```

Comment:
This tutorial of exporting model is not English.

Reply:
?

Reply:
My fault.